build.gradle

```diff
@@ -123,7 +120,11 @@ bower {
 
     /* Introduced by the Blue Ocean landing page */
     'ionicons' ('2.0.1') {
-        source '*'
+        source '**/*.css'
+        source '**/*.woff'
+        source '**/*.ttf'
+        source '**/*.svg'
+        source '**/*.eot'
```

Comment:
if we don't be very specific, something in gradle/awestruct tries to compile the less included in this package and it busts the build and boom 💥